internal/controller/promotionstrategy_controller.go

```diff
@@ -256,6 +256,12 @@ func (r *PromotionStrategyReconciler) createOrUpdatePreviousEnvironmentCommitSta
 	gvk := promoterv1alpha1.GroupVersion.WithKind(kind)
 	controllerRef := metav1.NewControllerRef(ctp, gvk)
 
+	// If there is only one commit status, use the URL from that commit status.
+	var url string
+	if len(previousCRPCSPhases) == 1 {
+		url = previousCRPCSPhases[0].Url
+	}
```

Comment:
Is there a good place to document this behavior?

Reply:
gating-promotions.md has a section about the previous environment commitstatus, that seems like an okay place to call it out.